mato.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = ">= 2.3"
+  spec.required_ruby_version = ">= 2.4"
```

Comment:
RuboCop has dropped to support Ruby 2.3. So mate also dropped it.
Ruby 2.3 is too old, so I think we can drop to support it with only a few impacts.